### ISSUE TITLE
Set the default number of samples based on heuristic

### ISSUE THREAD

**Issue by jlperla** [HighDimensionalEconLab/VarianceComponentsHDFE.jl#3]
From @rsaggio87 

The default value to log(K)/0.01 where K = # of person + firm fixed effects
this should return to  600 simulations when there is a 1,000,000 parameters being estimated. Our calibration in the paper shows that we these numbers we get back basically the “exact” solution

@paulcorcuera  Maybe you can clearly specify where to get those numbers of effects.  But @Alim-faraji 
- Maybe change teh https://github.com/HighDimensionalEconLab/VarianceComponentsHDFE.jl/blob/master/src/VarianceComponentsHDFE.jl#L57-L60 to be
```julia
        "--simulations"
            help = "number of simulations in the JLA algorithm.  If 0, defaults to 100 * log(#total fixed effect)"
            arg_type = Int
            default = 0
```

There are all sorts of ways to put in the default calculation. but right now everthing is in the `compute_whole` (which should eventually be torn apart).  So mabye the easiset is to just check if the value is 0 right before you need to do the draws.  You guys can discuss.

We get those numbers in this part of the code : https://github.com/HighDimensionalEconLab/VarianceComponentsHDFE.jl/blob/master/src/leave_out_correction.jl#L1055-L1056 

So basically the number we care about is J (firm effects)+N (person effects) , and use log(J+N)/0.01
